values.yaml

```diff
@@ -1,13 +1,17 @@
 # For detailed explanation of each of the configuration settings see
 # https://www.pomerium.io/reference/
 
+nameOverride: pomerium
```

Comment:
I'm not sure we want to override the name / full name in it's default configuration. Is this required for passing strict?

Reply:
Yes, unfortunately, it is required when running `helm lint --strict`.

We can change it to `""` if you think it's better.

Reply:
yes I think so too, I made the same comment on the helm repository actually :)